spec/flipper/expressions/now_spec.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.describe Flipper::Expressions::Now do
+  describe "#call" do
+    it "returns current time" do
+      expect(described_class.call.round).to eq(Time.now.round)
```

Comment:
I usually use expect be_within here for when the second passes. 

```suggestion
      expect(described_class.call).to be_within(2).of(Time.now.utc)
```